addons/sales_team/sales_team.xml

```diff
@@ -128,6 +126,17 @@
                                     </kanban>
                                 </field>
                             </page>
+                            <page string="More Info">
```

Comment:
Is the model and/or view limited to sale manager ? If yes it is redundant. If no, it is not redundant, because people could land on the view without going through the dropdown.